src/source-dpdk.c

```diff
@@ -104,6 +142,7 @@ typedef struct DPDKThreadVars_ {
     TmSlot *slot;
     LiveDevice *livedev;
     ChecksumValidationMode checksum_mode;
+    bool intr_en;
```

Comment:
`intr_en` means `interrupts_enabled` or something?

Reply:
yes, changed to `intr_enabled`
`intr` is an acronym also used in DPDK functions